covidfaq/routers/answers.py

```diff
@@ -1,12 +1,11 @@
-from typing import List, Optional
+from typing import List
```

Comment:
so, this part will spaw Xiang's model with your model (based on question asnwering), correct?

Reply:
yes